pkgs/by-name/di/disktui/package.nix

```
@@ -0,0 +1,23 @@
+{
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "disktui";
+  version = "1.2.0";
+  src = fetchFromGitHub {
+    owner = "Maciejonos";
+    repo = "disktui";
+    rev = "v${version}";
+    hash = "sha256-FDpdOpyvdU2Uw22am/Vkdls+s6ZdmodNt3WAQd8L53I=";
+  };
+  cargoHash = "sha256-CBSd/zeThyhmsaKx8Pg+u14QEQVq5nPLcRKet9n8WC8=";
+
+  meta = with lib; {
+    description = "TUI for disk management on Linux";
+    homepage = "https://github.com/Maciejonos/disktui";
+    license = licenses.mit;
+    maintainers = with maintainers; [ Inarizxc ];
+  };
```

`with` before attrsets (especially `with lib;`, _especially_ file-wide ones, _**especially**_ multiple ones) has been abused to death in the past and is now frowned upon[^1]. Just prepend `lib.` where necessary

[^1]: `with` before small scopes, like lists, is fine

```suggestion
  meta = {
    description = "TUI for disk management on Linux";
    homepage = "https://github.com/Maciejonos/disktui";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ Inarizxc ];
    platforms = lib.platforms.linux;
  };
```
and ofc limit the program to Linux (it doesn't need to evaluate on Darwin)